mods/cnc/rules/defaults.yaml

```diff
@@ -668,8 +668,8 @@
 	ActorPreviewPlaceBuildingPreview:
 		OverridePalette: placebuilding
 	SoundOnDamageTransition:
-		DamagedSounds: xplos.aud
-		DestroyedSounds: crumble.aud
+		DamagedSounds: xplobig4.aud
+		DestroyedSounds: crumble.aud, xplobig4.aud
```

Comment:
Did you intend for these both to play at the same time (which is what the remaster source suggests the original did)? This will currently pick one or the other, we will need to define a second instance of the trait if we want both at the same time.

Reply:
No they should not play at the same time for the sake of variety. A building either reports `crumble` or `xplobig4` in the original or remaster.